api/sys_plugins.go

```diff
@@ -244,6 +251,57 @@ func (c *Sys) RegisterPluginWithContext(ctx context.Context, i *RegisterPluginIn
 	return err
 }
 
+// RegisterPluginV2 wraps RegisterPluginWithContextV2 using context.Background.
+func (c *Sys) RegisterPluginV2(i *RegisterPluginInput) (*RegisterPluginResponse, error) {
```

Comment:
Could you expand a bit on why the this new versioned method is required? It seems to have the same signature as `RegisterPlugin`?

Reply:
@helenfufu thanks for the explanation. Do you think we could perhaps rename the method? Calling it v2 could be confusing in the future. Maybe `RegisterPluginAndCheck()` or `RegisterPluginDetailed()` or ...